clusterloader2/pkg/measurement/common/simple/metrics_for_e2e.go

```diff
@@ -97,7 +97,14 @@ func (m *metricsForE2EMeasurement) Execute(config *measurement.MeasurementConfig
 	}
 	grabMetricsFromKubelets = grabMetricsFromKubelets && strings.ToLower(provider) != "kubemark"
 
-	grabber, err := metrics.NewMetricsGrabber(config.ClientSets.GetClient(), nil, grabMetricsFromKubelets, true, true, true, false)
+	grabber, err := metrics.NewMetricsGrabber(
+		config.ClusterFramework.GetClientSets().GetClient(),
+		nil,
```

Comment:
NIT: Could you add comments explaining what each argument is, something like
...
grabMetricsFromKubelets,
true, /* do X */
false, / * useFeatureZ */

Reply:
Good idea. Done.